smart_room/app.py

```diff
@@ -0,0 +1,72 @@
+import os
```

Comment:
You can add a blank line after built-in package

Reply:
Fix in 1d4e9ef.